react-native-fbsdk-next.podspec

```diff
@@ -6,25 +6,25 @@ Pod::Spec.new do |s|
   s.version       = package['version']
   s.summary       = package['description']
   s.requires_arc  = true
-  s.author        = { 'dzhuowen' => 'dzhuowen@fb.com' }
+  s.author        = { 'thebergamo' => 'marcos@thedon.com.br' }
   s.license       = package['license']
   s.homepage      = package['homepage']
-  s.source        = { :git => 'https://github.com/facebook/react-native-fbsdk.git', :tag => "v#{package['version']}" }
+  s.source        = { :git => 'https://github.com/thebergamo/react-native-fbsdk-next.git', :tag => "v#{package['version']}" }
   s.platforms     = { :ios => "9.0", :tvos => "9.2" }
   s.dependency      'React'
 
   s.subspec 'Core' do |ss|
-    ss.dependency     'FBSDKCoreKit', '~> 8.1'
+    ss.dependency     'FBSDKCoreKit', '~> 9.0'
```

Comment:
For me ios-sdk should better be upgraded to at least `v9.0.1` because `autoInit` is removed from the ios-sdk in `v9.0.0` and there is a deadlock issue due to the removal.

See [facebook-ios-sdk changelog](https://github.com/facebook/facebook-ios-sdk/blob/master/CHANGELOG.md)